doc/source/security/apparmor.rst

```diff
@@ -0,0 +1,21 @@
+.. _apparmor:
+
+=====================================================================
+Enabling AppArmor 
+=====================================================================
+
+|Percona XtraDB Cluster| contains several AppArmor profiles which allows for easier maintenance. The ``mysqld`` profile is decoupled from the SST script profile. This option allows the introduction of other SST methods or scripts with their own profiles. The following profiles are:
+
+* An extended version of the |Percona Server| profile which allows the execution of SST script.
+
+* An xtrabackup-v2 SST script profile located in ``/etc/apparmor.d/usr.bin.wsrep_sst_xtrabackup-v2
+
+The ``mysqld`` profile allows the execution of the SST script in PUx mode with the /{usr/}bin/wsrep_sst_*PUx command. If the script applies the profile, if the script contains its own profile. If the script does not contain a profile the SST script runs in unconfined mode. If necessary, the system administrator can change the execution mode to Pix. This action causes a fallback to inherited mode in case the SST script profile is absent.
```

Comment:
```If the script applies the profile, if the script contains its own profile.```
Is this sentence OK?
If script AA profile related to the scrip is present, it is applied.

Reply:
Rewritten